### ISSUE TITLE
The library is old so I want you to update it.

### ISSUE THREAD

**Issue by famibee** [kevva/lpad-align#7]
Due to security issues, the library is out of date and I would like you to update it.
セキュリティの問題から、ライブラリが古いので更新して欲しいです。

| Lib | Now | Latest |
|---|---|---|
| get-stdin | ^5.0.1 | 9.0.0 |
| indent-string | ^3.0.0 | 5.0.0 |
| meow | ^3.3.0 | 10.0.1 |

https://david-dm.org/kevva/lpad-align

**Comment by Eskaan** [comment]
Yeah, same for me. The library is not using the latest version of meow, causing in two `ReDoS` warnings in my audit :(
It is a simple fix - just update the dependent and done. I will open a pull request for this in a few minutes to make it even more easy for you @kevva.
